crates/fluvio-connector-package/src/metadata.rs

```diff
@@ -313,19 +313,6 @@ fn validate_deployment(deployment: &Deployment, config: &ConnectorConfig) -> any
     Ok(())
 }
 
-fn validate_secrets(secrets: &Secrets, config: &serde_yaml::Value) -> anyhow::Result<()> {
```

Comment:
We should also probably delete `secrets` field and type from `ConnectorMetadata` as it is no longer being used.

Reply:
Removed in https://github.com/infinyon/fluvio/pull/3194/commits/9ccec8ba81ace408a4dae24599c054289ce212e9